src/ui/routes/admin.js

```diff
@@ -347,7 +347,6 @@ router.get('/auditlog_csv', mustBeAdminOrApproverMiddleware, function (req, res,
             outStream.write('Api;Application;Plan;Date (UTC);Activity;User;Email;Role\n');
             for (let i = 0; i < auditResponse.items.length; ++i) {
                 const item = auditResponse.items[i];
-                utils.processDisplayNames(item);
```

Comment:
is this audit log related ? why are u removing it ?

Reply:
Sure will add It again